lib/Dialect/FIRRTL/Ops.cpp

```diff
@@ -1024,7 +1024,21 @@ static LogicalResult verifyStdIntCast(StdIntCast cast) {
   // match.
   FIRRTLType firType;
   IntegerType integerType;
-  if ((firType = cast.getOperand().getType().dyn_cast<FIRRTLType>())) {
+  if (auto flippedType = cast.getOperand().getType().dyn_cast<FlipType>()) {
```

Comment:
This could be another place where `getPassiveType()` could be used.

Reply:
Moved getPassiveType into the conditionals.